packages/cli/src/commands/__tests__/commands.spec.ts

```diff
@@ -0,0 +1,140 @@
+import * as utils from '@stoplight/prism-http';
+import * as yargs from 'yargs';
+import { createMultiProcessPrism, createSingleProcessPrism } from '../../util/createServer';
+import mockCommand from '../mock';
+import proxyCommand from '../proxy';
+
+const parser = yargs.command(mockCommand).command(proxyCommand);
+
+jest.mock('../../util/createServer', () => ({
+  createMultiProcessPrism: jest.fn(),
+  createSingleProcessPrism: jest.fn(),
+}));
```

Comment:
`jest.mock('../../util/createServer');` overrides exported functions with `jest.fn()` for you. You don't need to supply implementation of module, the code inside `createServer` won't run.

Reply:
Oh cool, didn't know that. 8f2685e